blockly-rtc/server/http_server.js

```diff
@@ -23,14 +23,16 @@
 const http = require('http'); 
 const url = require('url');
 
-const sql = require('./db_utils');
+const Database = require('./Database').Database;
+const db = require('./db');
 
+const database = new Database(db);
 const PORT = 3001;
 
 http.createServer((req, res) => {
   const parsedUrl = url.parse(req.url, true);
   if (req.method === 'GET' && parsedUrl.pathname === '/api/events/query') {
-    sql.queryDatabase(parsedUrl.query.serverId)
+    database.queryDatabase(parsedUrl.query.serverId)
```

Comment:
I suggest renaming the function to `query` now that `database` is already in the name.

Reply:
Good point! Done.